Update README.md (#27)
AdaGold/hotel: README.md

- As a user of the hotel system...
  - I can access the list of all of the rooms in the hotel

>   - I can get the reservations of a room for a given date range

Reviewer: ```suggestion
  - I access the list of reservations for a specified room and a given date range
```
